bazel/spec-bundling/index_rjs.bzl

```diff
@@ -1,5 +1,5 @@
+load("@aspect_rules_esbuild//esbuild:defs.bzl", "esbuild")
 load("@devinfra//bazel/spec-bundling:spec-entrypoint.bzl", "spec_entrypoint")
-load("@devinfra_npm//bazel:esbuild/package_json.bzl", esbuild = "bin")
```

Comment:
If we are removing esbuild usage here, can we removed it from the deps in `//bazel:package.json`? It seems to be now controlled via the workspace import of `aspect_rules_esbuild`